main_test.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
IMO this should be named something different or live elsewhere as it's not actually testing anything in `main.go` like a unit test would, possibly calling this `integration_test.go` and leaving it in the `main` package would be sufficient.

Reply:
Good point, I'll rename 👍